v2/community/supertokens-core/saas-setup.mdx

```diff
@@ -10,89 +10,55 @@ hide_title: true
 import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs";
 import TabItem from '@theme/TabItem';
 
-# Managed Service
+# Set up Your SuperTokens Account 
 
-## Creating a development environment ✨
-- First, please [sign up](https://supertokens.com/auth)
-- Select the auth method you want to use and follow the guided steps to integrate with our frontend and backend SDK if you have not done this already
-<img alt="Integration with SuperTokens SDKs" width="500px" src="/img/saas/supertokens-core-quicksetup.png" />
-- Select a region and click the deploy button:
+:::info
+
+This guide assumes that you are using the **SuperTokens** managed service.
+If you want to self host the core service please check the [following guide](https://github.com/supertokens/supertokens-core#self-hosting).
```

Comment:
the link for self hosted should go to the self hosted setup section in the docs. The github readme doesn't explain how to do self hosted setup (but we can change it)